seisbench/models/base.py

```diff
@@ -457,22 +445,23 @@ def _get_versions_from_files(name, files):
         return sorted([config[prefix_len:] for config in configs])
 
     @classmethod
-    def load(cls, path):
+    def load(cls, path, version_str=None):
         """
         Load a SeisBench model from local path.
 
         For more information on the SeisBench model format see py:func:`save`.
 
         :param path: Define the path to the SeisBench model.
         :type path: pathlib.Path ot str
+        :param version_str: Version string of the model. If none, no version string is appended.
+                            The version string can alternatively be provided by directly providing it with the `path`,
+                            e.g., my_weight.json.v1. The `version_str` will take precedence over the `path` if both
+                            indicate a version.
+        :type version_str: str, None
```

Comment:
Slightly confused about the docstring for this and what it indicates (from my understanding anyway I could be missing something!). 

As the ```path``` parameter is typically the base path and suffixes are then added to this, what would be the point in saving out a .json.v1 file only. If you can explicitly define the full path including version strings, then you have to also inlcude whether you are saving out a .pt or .json file. From my side, it would just be cleaner to always have the user define any ```version_str``` to load. Otherwise if defaults to the standard lookup (without a version suffix).